.stylelintrc.json

```diff
@@ -7,4 +7,4 @@
     "csstree/validator": true
   },
   "ignoreFiles": ["build/**", "dist/**", "**/reset*.css", "**/bootstrap*.css", "**/*.js", "**/*.jsx"]
-}
\ No newline at end of file
+}
```

Comment:
- [ ] I think it's better if we use the README file from [Microverse Template](https://github.com/microverseinc/readme-template). Although the one from CRA is not incorrect, it does seem generic and doesn't document the project in a very professional way. I do recommend as well including the scripts from this document in the other.

[OPTIONAL] Make sure that when you open a pull request, all of the changes required from Microverse are included in this Pull Request, this facilitates the review process as Code Reviewers are able to comment exactly on the pieces of code that might need a change.

Reply:
Thank you for your observation. I have indeed updated my README according to your preferences

Reply:
I also did the [OPTIONAL] requirement. I thank you again